src/easier68k/core/util/conversions.py

```diff
@@ -1,39 +1,38 @@
 # Conversion utils
 
 
-def word_to_long(word: int) -> int:
+def to_word(word: int) -> int:
     """
-    >>> hex(word_to_long(0x12341234))
+    >>> hex(to_word(0x12341234))
```

Comment:
I'd pick a better use case than this, like `0x12345678`, just because here we can't tell which half of the long it's taking for the word.